biofefi/options/enums.py

```diff
@@ -81,6 +81,8 @@ class ProblemTypes(StrEnum):
     Auto = "auto"
     Classification = "classification"
     Regression = "regression"
+    Binary_Classification = "binary_classification"
+    Multi_Classification = "multi_classification"
```

Comment:
⛏️ Can we make the field names PascalCase like everything else. No underscores. The bits in the double quotes are fine though